node/src/components/rpc_server/rpcs/info.rs

```diff
@@ -225,3 +236,98 @@ impl RpcWithoutParamsExt for GetStatus {
         .boxed()
     }
 }
+
+/// A single change to a validator's status between two eras.
+#[derive(Serialize, Deserialize, Debug, JsonSchema)]
+#[serde(deny_unknown_fields)]
+pub struct JsonEraChange {
+    /// The era in which the change occurred.
+    era_id: EraId,
+    /// The change in validator status.
+    validator_change: ValidatorChange,
+}
+
+impl JsonEraChange {
```

Comment:
Maybe `JsonValidatorStatusChange` or `JsonValidatorChange` would be a better name?

`JsonEraChange` sounds like information about the whole change from one era to the next, not like it's specific to one validator.

Reply:
Addressed in [`d0417f9`](https://github.com/darthsiroftardis/casper-node/pull/12/commits/d0417f96c45978441e9e0c0e615610862fca246c).